### PR TITLE
feat: improve group users select search

### PR DESCRIPTION
**Pull request by nunogois**
https://linear.app/unleash/issue/2-195/groups-improve-search

### CODE REVIEW

File: frontend/src/component/project/ProjectAccess/ProjectAccessAssign/ProjectAccessAssign.tsx [hunk 1]
```diff
@@ -88,6 +88,9 @@ const StyledRoleOption = styled('div')(({ theme }) => ({
     },
 }));
 
+const caseInsensitiveSearch = (search: string, value?: string) =>
+    Boolean(value?.toLowerCase()?.includes(search.toLowerCase()));
+
```
Comment (gastonfournier) [inline frontend/src/component/project/ProjectAccess/ProjectAccessAssign/ProjectAccessAssign.tsx:93]: I wonder if this should be in some kind of library

**Review by gastonfournier — APPROVED** [review]
One minor detail, but it looks fine :+1:

File: frontend/src/component/project/ProjectAccess/ProjectAccessAssign/ProjectAccessAssign.tsx (hunk 1 above)
Comment (nunogois) [reply frontend/src/component/project/ProjectAccess/ProjectAccessAssign/ProjectAccessAssign.tsx]: How about https://github.com/Unleash/unleash/pull/1988/commits/5a1250a8f7f8a4363619cd2d26e1237d0d64a033?

Comment (gastonfournier) [reply frontend/src/component/project/ProjectAccess/ProjectAccessAssign/ProjectAccessAssign.tsx]: Amazing :rocket:

**Pull request merged**